src/themes/masschallenge/modules/modal.overrides

```diff
@@ -17,33 +17,94 @@
 }
 
 .ui.modal > .teal.segment {
-  background-color:  #b8dde4 !important;
+  background-color: #b8dde4 !important;
```

Comment:
Why does this require an !important flag?

Reply:
This is not part of this ticket. I was fixing spacing on this